modules/date.js

```diff
@@ -0,0 +1,15 @@
+/* eslint-disable no-undef */
+const webDate = document.getElementById('date');
+const date = () => {
+  window.setInterval(
+    () => {
+      webDate.innerHTML = `<p>${luxon.DateTime.local().toLocaleString(
+        luxon.DateTime.DATETIME_MED_WITH_SECONDS,
+      )}</p>`;
+    },
+    1000,
+    this,
+  );
+};
+
+export default date;
```

Comment:
- [ ] Luxon Package should be added as a .js file to the project (in full or minified version)
- [ ] Module is imported as follows `import { DateTime } from "luxon";`